code/drasil-example/Drasil/SSP/GenDefs.hs

```diff
@@ -192,14 +192,14 @@ effNormFDeriv = mkDerivNoHeader [foldlSent [
   sParen (makeRef2S assumpPSC) `sC` S "the resulting", plural force,
   S "are expressed per", phrase metre, S "in", phraseNP (the zDir)]]
 
---
-normShrR :: RelationConcept
-normShrR = makeRC "normShrR"
-  (nounPhraseSP "interslice normal and shear force proportionality")
-  nmShrRDesc nmShrRRel -- genDef5Label
+-- 
+
+normShrR :: QDefinition 
+normShrR = mkQuantDef' normToShear  (nounPhraseSP "interslice normal and shear force proportionality") nmShrRRel
```

Comment:
I think this should be defining `intShrForce`, not `normToShear`. Does it still break stable if you try this instead?

Reply:
okay i just made the change, i think it should work, the first time i tried it was without the noun phrase and it caused errors, so i didn't think to go back to it, but it should be better now.

Reply:
Hmm, it looks like it still causes breakage of `stable/`. Where is the breakage located if you don't give the `mkQuantDef` a custom term? We might just need a custom term for the gd instead.